tests/components/flo/test_services.py

```diff
@@ -67,3 +67,17 @@ async def test_services(hass, config_entry, aioclient_mock_fixture, aioclient_mo
     )
     await hass.async_block_till_done()
     assert aioclient_mock.call_count == 12
+
+    # test calling with a string value to ensure it is converted to int
```

Comment:
Would there be an exception if sleep weren't an int?  Else I can't see this test being relevant.

Reply:
It is going to raise `voluptuous.error.MultipleInvalid: expected int for dictionary value @ data['sleep_minutes']` if it is not coercible.

It would be nice to have a test for that